docs/src/main/paradox/events-by-tag.md

```diff
@@ -1,76 +1,52 @@
+has been missed. Upon detection the query will search back in time for the missed events.
 
-The size of the of the batch is controlled via `max-message-batch-size`. Cassandra imposes a limit on the serialized size
-of a batch, currently 50kb by default. The larger this value the more efficient the tag writes will be but care must 
-be taken not to have batches that will be rejected by Cassandra. Two other cases cause the batch to be written before the batch size is reached:
 
-* Periodically: By default 250ms. To prevent eventsByTag queries being too out of date.
-* When a starting a new timebucket, which translates to a new partition in Cassandra, the events for the old timebucket are written.
+To detect missed events without receiving another event for the same persistence id a periodic back track is done
+to find events. This is configured with:  
 
+```
+cassandra-query-journal.events-by-tag.back-track {
+      # Interval at which events by tag stages trigger a query to look for delayed events from before the
+      # current offset. Delayed events are also searched for when a subsequent event for the same persistence id
+      # is received. The purpose of this back track is to find delayed events for persistence ids that are no
+      # longer receiving events. Depending on the size of the period of the back track this can be expensive.
+      interval = 1s
+
+      # How far back to go in time for the scan. This is the maxinum amount of time an event can be delayed
+      # and be found without receiving a new event for the same persistence id. Set to a duration or "max" to search back
+      # to the start of the previous bucket which is the furthest a search can go back.
+      period = 5s
+}
+```
 
-## Consistency
 
-Event by tag queries are eventually consistent, i.e. if a persist for an event has completed
-is does not guarantee that it will be visible in an events by tag query, but it will eventually.
-
-Order of the events is maintained per persistence id per tag by maintaining a sequence number for tagged events per persistence id.
-This is in addition to the normal sequence nr kept per persistence id.
-This additional sequence number is used to detect missing events and trigger searching back. However this only works if
-a new event for the same persistenceId is encountered. If your usecase involves many persistenceIds with only a small 
-number of events per persistenceId then the `eventual-consistency-delay` is essential to mitigate the risk of missing
-delayed events in live queries.
+The default `eventual-consistency-delay` is set to `5s` which is high as it is the safest. This can be set much lower 
+for latency sensitive applications and missed events will be detected via the above mechanisms. The downside is that
+the search for missing events is an expensive operation and with a high eventual consistency delay it won't happen without exceptional delays, 
+with a low value it will happen frequently.
```

Comment:
I think we should have a section about how to change the configuration for as low latency as possible, and explaining any tradeoffs. This seems to be a repeated question.

Suggestion of such settings:
```
cassandra-journal.events-by-tag.flush-interval = 0s
cassandra-journal.pubsub-notification = on
cassandra-query-journal.events-by-tag.eventual-consistency-delay = 50ms
cassandra-query-journal.refresh-interval = 2s
```

In my testing of with the cqrs sample that will keep the latency below 100 ms (typically around 70ms). I have only tested with one Cassandra node locally, so I don't know if that 50 ms is too low and would trigger backtracking in a real cluster.

Reply:
I've added a section for this and will update the values if anything comes up from the testing i'll do in AWS